### ISSUE TITLE
Request: add event detail to panZoom 

### ISSUE THREAD

**Issue by asdf23** [svgdotjs/svg.panzoom.js#89]
After using panZoom, I would like to know what to feed into .zoom( so that later I can programmatically return to this position.

I hope that makes since. I'll try to describe my workflow...

I have some SVG Art, I want to animate it by focus on specific areas over time. I cannot quite get correct zoom using zoom(. Instead I want to use panZoom, find the right spot then get the correct arguments to later feed into .zoom method. I do not see a way to do this currently.

**Comment by Fuzzyma** [comment]
If you want to return to exactly the same zoom, don't use zoom. It will only zoom the canvas to a level into a point and will probably not do what you want. Just save the `svg.viewbox()` instead and later set it back. This will give you the exact view

**Comment by asdf23** [comment]
Thanks for the tip, I was headed down that route.
I don't understand zoom, I can't seem to run .zoom(2,50,50) then run .zoom(2, 60,60) but I can if I change the zoom factor.
Also I can't really zoom anywhere I want. I don't know if it's my svg document or something I'm not understanding...

**Comment by asdf23** [comment]
addendum: I don't know if you or anyone is familiar with Impress.js: https://github.com/impress/impress.js/ .
tldr;  JavaScript flys the camera around a html document that is presented in 3d form. 

I was hoping to use this zoom feature like that. Set a svg document somewheres and zoom the camera while also moving the camera around to present points. Those points begin determined by some other logic.

**Comment by Fuzzyma** [comment]
Zoom will NOT center the point you passs to it. It will zoom INTO that point. That means, the point you pass is the point which does not move in the zoom process. Just imagine you scrolling your mouse to zoom and the point where your mouse is does not move away because you zoom into that point.

That also means that changing only the point but not the zoom level will do nothing at all. The functionality you want is probably better done with viewbox since you want to pan around as well
